.gitignore

```diff
@@ -63,6 +63,7 @@ cpuprofile
 tr1d1um
 .ignore
 .vscode
+.yaml
```

Comment:
i added a new .yaml folder for myself for running tests with yaml file that has secrets in it so i don't accidentally push that file

Reply:
can you use `.dev/*` instead?
```suggestion
```